rules/evasion/decrypt-eval.yara

```diff
@@ -1,14 +1,24 @@
 import "math"
 
-rule python_exec_near_enough_decrypt : critical {
+rule python_exec_eval_one_line : critical {
+  meta:
+    description = "Evaluates code from encrypted content on a single line via exec or eval"
+  strings:
+    $eval_decrypt_one_line = /evan\s{0,32}\(.{0,32}decrypt/ ascii wide
```

Comment:
typo?

Reply:
Ope, yep!

Reply:
Fixed in [`6a2e34c` (#340)](https://github.com/chainguard-dev/bincapz/pull/340/commits/6a2e34c15d71d8ef16b4dc2bab465e426662206c).